ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/slider/SliderScreen.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.android.wearable.composeadvanced.presentation.ui.slider
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.wear.compose.material.InlineSlider
+
+/**
+ * Displays a Slider, which allows users to make a selection from a range of values.
+ * https://developer.android.com/reference/kotlin/androidx/wear/compose/material/package-summary#InlineSlider(kotlin.Float,kotlin.Function1,kotlin.Int,androidx.compose.ui.Modifier,kotlin.Boolean,kotlin.ranges.ClosedFloatingPointRange,kotlin.Boolean,kotlin.Function0,kotlin.Function0,androidx.wear.compose.material.InlineSliderColors)
+ */
+@Composable
+fun SliderScreen(
+    displayValue: Int,
+    onValueChange: (Int) -> Unit
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(top = 10.dp),
+        verticalArrangement = Arrangement.Center
+    ) {
+        InlineSlider(
+            value = displayValue,
+            onValueChange = onValueChange,
+            valueProgression = 0..10,
```

Comment:
The slider here has a range of `0` to `10`, while the range for the stepper is `1` to `10`.

I'm not sure if that's intentional, but it also brings up something interesting:

What should the behavior be if the `value` falls outside the `valueProgression`?

Right now, if the value is `0` for the stepper of range `1..10`, then clicking on the `-` button brings the value to `1`, and clicking on the `+` button brings it to `2`. Clicking nothing leaves it at an invalid value of `0`.

Reply:
I updated the slider valueProgression range to be 1..10

Reply:
I suggest adding this valid question in the feature request that Yuri and I proposed. The bug id is 219740135